client/components/product-expiration/index.tsx

```diff
@@ -0,0 +1,65 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+
+interface Moment {
+	isValid: () => boolean;
+	format: ( arg0: string ) => string;
+	diff: ( arg0: Date ) => number;
+}
+
+export interface Props {
```

Comment:
Do we really need to export it?

Reply:
Nyet! Fixed in 16feb54d14598303f399cc90336be44c634d157b